### DISCUSSION TITLE
error: RPC failed; HTTP 500 curl 22 The requested URL returned error: 500

### DISCUSSION THREAD

**Discussion by edgars-cirulis** [community/community#114524] [Programming Help] [answered]
Hello there. I've started encountering this error message while using the HTTPS protocol:
warning: redirecting to https://git.codelinaro.org/clo/la/platform/vendor/qcom/opensource/audio-kernel-ar.git/
error: RPC failed; HTTP 500 curl 22 The requested URL returned error: 500
fatal: expected 'acknowledgments'

The same issue persists on GitHub. I am able to fetch via SSH protocol but not HTTPS.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Edgars-Cirulis ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by mecodeatlas** [comment, marked as answer]
Hey @Edgars-Cirulis! 👋 

We're going through our unanswered discussions, I know it has been a minute, but I wanted to check and see if you were able to get this solved?

**Comment by AKUMON12** [comment]
How can I resolve this?

$ git push -u origin main
remote: Internal Server Error
fatal: unable to access 'https://github.com/AKUMON12/skillstest_ict_congress.git/': The requested URL returned error: 500
